source/installation/raspberrypi.markdown

```diff
@@ -10,6 +10,10 @@ Included sections for this page is located under source/_includes/installation
 {% assign board = "Raspberry Pi" %}
 {% assign installation_media = "SD card" %}
 
+### SD Card usage
```

Comment:
1) This does not apply to raspberry pi only, this can be done with all OS installations, the note should be in the shared operating_system.md file
2) This links to developer documentation, before linking that section should be moved to the common task section for OS installations.
3) How to do this is currently changing (see https://github.com/home-assistant/supervisor/pull/3120)

Reply:
You are right, good to know changes are coming, i will close the PR.